### ISSUE TITLE
[Telemetry] Add telemetry for the auto-close prompt

### ISSUE THREAD

**Issue by Amejia481** [mozilla-mobile/fenix#22170]
https://mozilla-hub.atlassian.net/browse/FNXV2-18283

## Description & Product Manager / Data Scientist User Story
Probes to implement:
Telemetry suggestions for the auto-close prompt:

- Impressions - Event
- Click “Turn on” - Event
- Dismiss prompt - Event

## What questions will you answer with this data?
Help to understand how user user the new auto-close prompt on the inactive tab section.

## Acceptance Criteria
- [ ] ENG files a [DS JIRA](https://jira.mozilla.com/projects/DO/issues/DO-228?filter=allopenissues) request outlining their methodology.
- [ ] DS sign off on instrumentation methodology addressing product questions.
- [ ] Event pings can be queried via re:dash
- [ ] Event pings can be queried via amplitude
- [ ] We are sending telemetry events for the actions listed in the requirements
- [ ] We have documented the telemetry
- [ ] We have asked a data steward to [review](https://github.com/mozilla/data-review/blob/master/request.md) the telemetry

**Comment by AndiAJ** [comment]
Hi @Amejia481 , the events aren't generated on Nightly 95.0a1 from 11/02 using a Google Pixel 3a (Android 11)
I've also tried force sending the event type pings but no luck.

Please review and share your thoughts. ☺️ 
I'll remove the QA needed label.

**Comment by Amejia481** [comment]
Thanks @AndiAJ !

**Comment by Amejia481** [comment]
@AndiAJ  we landed a fix, would you mind re-testing? :)

**Comment by AndiAJ** [comment]
Hi, verified as fixed on the latest Nightly 96.0a1 from 11/11 using a Google Pixel 3a (Android 11)

✔️ Seen the auto-close dialog for inactive tabs - Event type ping a82e6e42-2566-452b-8b4d-7c43d86b8d1d
```
{
          "timestamp": 5658,
          "category": "tabs_tray",
          "name": "auto_close_seen"
        }
```

✔️ Dismissed the auto-close dialog for inactive tabs - Event type ping a82e6e42-2566-452b-8b4d-7c43d86b8d1d
```
        {
          "timestamp": 7938,
          "category": "tabs_tray",
          "name": "auto_close_dimissed"
        }
```
✔️   Clicked the turn-on auto-close button for inactive tabs- Event type ping a7c9bb06-ecd3-4d57-b411-d19fbd44a199
```
        {
          "timestamp": 8501,
          "category": "tabs_tray",
          "name": "auto_close_turn_on_clicked"
        }
```
[Logcat](https://github.com/mozilla-mobile/fenix/files/7520586/clsintb.txt)
